### PR TITLE
Adds shotgun bandoliers

### PR DESCRIPTION
**Pull request by dannno**
bandoliers, they go on your belt
they hold 6 shotgun slugs

can someone else add some to armory/bar backroom because i'm too retarded to map

### CODE REVIEW

**Comment by WJohn** [comment]
It'd be nice if all ammo types had boxes, honestly. But this is cool too :+1:

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -173,4 +173,25 @@
...
+		/obj/item/ammo_casing/shotgun/beanbag,
+		/obj/item/ammo_casing/shotgun/improvised,
+		/obj/item/ammo_casing/shotgun/stunslug,
+		/obj/item/ammo_casing/shotgun/meteorshot,
+		/obj/item/ammo_casing/shotgun/pulseslug,
+		/obj/item/ammo_casing/shotgun/incendiary,
+		/obj/item/ammo_casing/shotgun/frag12,
+		/obj/item/ammo_casing/shotgun/incendiary/dragonsbreath,
+		/obj/item/ammo_casing/shotgun/techshell,
+		/obj/item/ammo_casing/shotgun/dart
```
Comment (paprka) [inline code/game/objects/items/weapons/storage/belt.dm]: ...you don't have to put all of these ammo types in here, /obj/item/ammo_casing/shotgun already does that, they're children of it

Comment (dannno) [inline code/game/objects/items/weapons/storage/belt.dm]: good to know

**Comment by Fox-McCloud** [comment]
Remind me again why there aren't just boxes of shotgun shells on TG? There's already a fire-rate limitation imposed on shotguns by virtue of how you fire them; is it really necessary to also make them incredibly slow to reload?

It would kinda crush the point of this PR if there were any, but it's still something I always wondered about.

**Comment by WJohn** [comment]
Well shotguns have by far the most diverse ammo types in the game. They can have 3-tile wide knockdown shots, "incendiary" shots that set the whole hallway on fire, are literally capable of firing meteors, have explosive slugs, and huge metal slugs that are equivalent to a .357 round. They pay for this with their slow reload and difficult pumping, but this can be handled by loading up your combat shotgun before a fight and spamming the use key to get a decent fire rate.

**Comment by dannno** [comment]
IMO shotguns work far better without boxed ammo anyways.

**Comment by dannno** [comment]
ps. merge :)

**Comment by Razharas** [comment]
Looks ok, maybe give one to barkeep?

**Comment by dannno** [comment]
yeah I'm too dumb 2 map tho : - D

**Comment by paprka** [comment]
Shotguns can get shell boxes later for convenience

**Comment by dannno** [comment]
merg

**Comment by Fox-McCloud** [comment]
Barkeep should probably start with one of these full instead of his backpack having the ammo.

**Comment by paprka** [comment]
^ this

**Pull request merged**